lib/install/planbuilder.go

```diff
@@ -602,7 +602,7 @@ func (i *Installer) GetPlanBuilder(cluster ops.Site, op ops.SiteOperation) (*Pla
 		},
 		InstallerTrustedCluster: trustedCluster,
 	}
-	err = addResources(builder, cluster.Resources, i.Config.RuntimeResources)
+	err = addResources(builder, cluster.Resources, i.Config.RuntimeResources, i.Config.ClusterResources)
```

Comment:
`cluster.Resources` is the only case of []byte-typed resource left - it is used for (and with) the Gravity `cluster` resource.